recipes/repl-builtin-modules/src/workflow.ts

```diff
@@ -0,0 +1,438 @@
+                            const standaloneReferences = rootNode.findAll({
+                                rule: {
+                                    pattern: "_builtinLibs"
+                                }
+                            });
+
+                            for (const ref of standaloneReferences) {
+                                const parent = ref.parent();
+                                if (parent && parent.kind() !== "named_imports" && parent.kind() !== "member_expression") {
+                                    edits.push(ref.replace("builtinModules"));
+                                }
+                            }
+                        }
+                        hasChanges = true;
+                    }
+                }
+            }
+        }
+
+        // Handle default imports like: import repl from 'node:repl'
+        // Handle namespace imports like: import * as repl from 'node:repl'
+        if (!namedImports) {
+            // Look for default or namespace imports that use repl.builtinModules or repl._builtinLibs
+            const importClause = statement.find({
+                rule: {
+                    kind: "import_clause"
+                }
+            });
+
+            if (importClause) {
+                let importIdentifier = null;
+
+                // Check for default import
+                const defaultImport = importClause.find({
+                    rule: {
+                        kind: "identifier"
+                    }
+                });
+
+                // Check for namespace import
+                const namespaceImport = importClause.find({
+                    rule: {
+                        kind: "namespace_import"
+                    }
+                });
+
+                if (defaultImport) {
+                    importIdentifier = defaultImport;
+                } else if (namespaceImport) {
+                    const namespaceIdentifier = namespaceImport.find({
+                        rule: {
+                            kind: "identifier"
+                        }
+                    });
+                    if (namespaceIdentifier) {
+                        importIdentifier = namespaceIdentifier;
+                    }
+                }
+
+                if (importIdentifier) {
+                    const varName = importIdentifier.text();
+
+                    // Find all member expressions using this variable with builtinModules or _builtinLibs
+                    const builtinModulesExpressions = rootNode.findAll({
+                        rule: {
+                            pattern: `${varName}.builtinModules`
+                        }
+                    });
+
+                    const builtinLibsExpressions = rootNode.findAll({
+                        rule: {
+                            pattern: `${varName}._builtinLibs`
+                        }
+                    });
```

Comment:
```suggestion
                    const allUsages = rootNode.findAll({
                        rule: {
                        any: [
                          {
                              pattern: `${varName}.builtinModules`
                          }, 
                          {
                              pattern: `${varName}._builtinLibs`
                          }
                    });
```

I think `any` can be used here to simplified it